### ISSUE TITLE
vsearch in R markdown issue

### ISSUE THREAD

**Issue by js3110** [torognes/vsearch#510]
When I try to run vsearch in an R markdown file using bash I am getting the error vsearch: command not found

In the console if i run system('vsearch -version') i get:
sh: vsearch: command not found
Warning message:
In system("vsearch -version") : error in running command

But in the terminal on R, vsearch -version gives:
vsearch v2.5.1_macos_x86_64, 8.0GB RAM, 4 cores
https://github.com/torognes/vsearch

Rognes T, Flouri T, Nichols B, Quince C, Mahe F (2016)
VSEARCH: a versatile open source tool for metagenomics
PeerJ 4:e2584 doi: 10.7717/peerj.2584 https://doi.org/10.7717/peerj.2584

I have installed it but it isn't working in the markdown file and I don't know how to fix it.

**Comment by frederic-mahe** [comment]
> When I try to run vsearch in an R markdown file using bash

I don't think you can run a Rmarkdown file directly in a bash terminal. You would need to extract the bash chunks to a bash script first. 

`vsearch` is not a R package, how did you install it?

You seem to be working on macOS, could you please try to run `which vsearch` in a Terminal? That would help to locate the `vsearch` binary on your system.

**Comment by js3110** [comment]
Sorry, when I said using bash I mean that the chunk uses {bash} not that its directly in a bash terminal.

I installed vsearch on my mac, not on R, using instructions from https://astrobiomike.github.io/bash/installing_tools
Running which vsearch returns nothing in my Terminal

**Comment by frederic-mahe** [comment]
did you find a way to fix that issue?

Also, I suggest to install a more recent version of `vsearch` if you can. The latest is [version 2.22.1](https://github.com/torognes/vsearch/releases/download/v2.22.1/vsearch-2.22.1-macos-x86_64.tar.gz).

**Comment by js3110** [comment]
Yes, I reinstalled vsearch and made sure that it was in the same path as my other installations and now its wokring. For some reason the first time I installed it something didnt work so it wasn't actually installed.
Thanks!

**Comment by frederic-mahe** [comment]
good! thank you for trying `vsearch`
